distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -1416,6 +1493,220 @@ type Builder
            changes to the builder will not affect the returned vector.
         Vector.from_polyglot_array self.java_builder.toArray
 
+## PRIVATE
+type Build_Style_Builder
```

Comment:
This new Builder only has the methods needed by the `new_builder` calls that have been converted to `build`. More will be added as needed.